sanic/config.py

```diff
@@ -83,6 +83,8 @@ def from_pyfile(self, filename):
         except IOError as e:
             e.strerror = 'Unable to load configuration file (%s)' % e.strerror
             raise
+        except Exception as e:
+            raise
```

Comment:
Is this just to raise the "unknown" `Exception` instance `e`? In that case, wouldn't be better to `raise e` or `raise Exception from e` ?